test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetListCommandTest.cs

```diff
@@ -97,7 +97,7 @@ public void ListCommand_WithUserSpecifiedSource()
 
                 // Assert
                 Assert.Equal(0, result.ExitCode);
-                var output = result.Output;
+                var output = RemoveDeprecationWarning(result.Output);
```

Comment:
I see you have a lot of tests removing the deprecation warning from the output, can we add a test asserting that the list command is deprecated and search should be used instead?